adapters/BasicAdapter/src/main/kotlin/me/gabber235/typewriter/entries/event/PlayerDeathEventEntry.kt

```diff
@@ -5,17 +5,16 @@ import me.gabber235.typewriter.adapters.Entry
 import me.gabber235.typewriter.entry.*
 import me.gabber235.typewriter.entry.entries.EventEntry
 import me.gabber235.typewriter.utils.Icons
-import org.bukkit.entity.EntityType
 import org.bukkit.entity.Player
+import org.bukkit.event.entity.EntityDamageEvent.DamageCause
 import org.bukkit.event.entity.EntityDeathEvent
-import java.util.*
 
 @Entry("on_player_death", "When a player dies", Colors.YELLOW, Icons.SKULL_CROSSBONES)
 class PlayerDeathEventEntry(
 	override val id: String = "",
 	override val name: String = "",
 	override val triggers: List<String> = emptyList(),
-	val entityType: EntityType = EntityType.PLAYER,
+	val deathCause: DamageCause = DamageCause.CUSTOM,
```

Comment:
Maybe make this optional. Then users can choose if the death cause matters for their situation.